### ISSUE TITLE
Remove themes from Text teaser

### ISSUE THREAD

**Issue by NickHaggerty1** [equinor/energyvision#2173]

**Comment by NickHaggerty1** [comment]
We have created a couple of themes which fail Wcag colour checking in the new Text teaser. 
Could we please remove/replace:
Mid orange with Red and Mid green with Red. I would like to keep these background colour options with the default black text colour instead.

**Comment by NickHaggerty1** [comment]
@millianapia Great, thanks approved
